moto/elbv2/models.py

```diff
@@ -1080,6 +1087,90 @@ def create_target_group(self, name: str, **kwargs: Any) -> FakeTargetGroup:
                 "HttpCode must be like 200 | 200-399 | 200,201 ...",
             )
 
+        if target_type in ("instance", "ip", "alb"):
+            for param in ("protocol", "port", "vpc_id"):
+                if not kwargs.get(param):
+                    param = "VPC ID" if param == "vpc_id" else param.lower()
+                    raise ValidationError(f"A {param} must be specified")
+
+        if vpc_id := kwargs.get("vpc_id"):
```

Comment:
We're still supporting Python 3.7, so the `:=` operator will not work unfortunately

Reply:
Updated